cmd/internal/cli/build_linux.go

```diff
@@ -425,7 +429,8 @@ func runBuildLocal(ctx context.Context, cmd *cobra.Command, dst, spec string) {
 				SandboxTarget:     sandboxTarget,
 				// Only perform a build with the host DefaultPlatform at present.
 				// TODO: rework --arch handling for remote builds so that local builds can specify --arch and --platform.
-				Platform: *dp,
+				Platform:    *dp,
+				ReqAuthFile: reqAuthFile,
```

Comment:
Perhaps, on thinking about it... this should be `DockerAuthFIle` in types.Options ... so it groups with the other things thare are specific to builds from docker sources?

I don't think we use it except for builds from docker sources?